src/js/themes/base.js

```diff
@@ -299,6 +299,7 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       primary: {
         // color: { dark: undefined, light: undefined }
       },
+      disabled: undefined,
```

Comment:
I think this should be commented out, like the other undefined values in this file.

Reply:
done!